apps/opik-backend/src/main/java/com/comet/opik/domain/CommitGenerator.java

```diff
@@ -0,0 +1,14 @@
+package com.comet.opik.domain;
+
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+
+import java.util.UUID;
+
+@UtilityClass
+class CommitGenerator {
+
+    public String generateCommit(@NonNull UUID id) {
```

Comment:
I understand the observation that nothing is being generated there, but we decided that today, the `commit` (I think it's better to call commit because this is how the user will be referring to it) is "generated", however this change so what about `getCommit`